### DISCUSSION TITLE
button  after triggering deleted :-) input field with length var.

### DISCUSSION THREAD

**Discussion by xverbatim1971** [mProjectsCode/obsidian-meta-bind-plugin#163] [Ideas]
hello,

first of all a big praise and thanks to the developer :-)
great plugin.

I have a few suggestions:

it would be nice if you could create a button that would disappear after triggering, i.e. be deleted :-) or hide.

great things would be possible :-)

...
a suggestion for the input fields would be that you could set the length of the input field via a length variable.
...

best regards
tom

**Comment by mProjectsCode** [comment]
Regarding the length, do you mean a max imput length? If so the [limit](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/reference/inputfieldarguments/limit/) argument might be what you are looking for.

Regarding the buttons. Could you please open a feature request under the issues tab for that?

**Reply by xverbatim1971** [reply]
"Regarding the length, do you mean a max imput length? If so the [limit](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/reference/inputfieldarguments/limit/) argument might be what you are looking for."


No, I don't mean the limit function, which already exists.

it's about the direct visual length of the input field...

it would be nice if you could set this...
80px
100px or whatever

"Regarding the buttons. Could you please open a feature request under the issues tab for that?"
ok, I hope that I have moved this correctly.
my english is very bad :-)

**Reply by mProjectsCode** [reply]
you can do that via a CSS snippet and the [class](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/reference/inputfieldarguments/class/) argument.

**Reply by xverbatim1971** [reply]
is there perhaps a css example file or example code for the exact application?

**Reply by mProjectsCode** [reply]
no, but this might help https://help.obsidian.md/Extending+Obsidian/CSS+snippets

**Reply by xverbatim1971** [reply]
thanks for the answer.

unfortunately the syntax for an input field or area field completely overwhelms me with a css command :-(

thanks anyway !
